tools/data-handler/src/containers/project.ts

```diff
@@ -190,6 +190,7 @@ export class Project extends CardContainer {
           key: entry.name,
```

Comment:
Just a note to self: It would make a code slightly simpler, if there would be some sort of `static` key creator that wouldn't need to set the empty arrays.

E.g. `const card = Defaults.card(name, path);` would create 
```
{
  key: name,
  path: path, 
  children: [],
  attachments: []
}
```